src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -1204,5 +1212,32 @@ private void PutConnectionInPool()
 
             _pool.PutConnection(this);
         }
+
+        private struct ValueStringBuilder
+        {
+            public char[] Chars;
+            public int Offset;
```

Comment:
Well, it is a start index :)  It's the start of the next location available for writing, which just so happens to also be the current length because of the chars already written starting at 0.  But, eh, I can change it.